docs/core_docs/docs/integrations/chat/novita.mdx

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_label: Novita
+---
+
+import CodeBlock from "@theme/CodeBlock";
+
```

Comment:
You shouldn't have this and the notebook above - the notebook compiles into an `.mdx` file

Reply:
Removed this file